platforms/documentation/docs/src/docs/userguide/authoring-builds/tasks/more_about_tasks.adoc

```diff
@@ -17,7 +17,11 @@
 
 A task represents some *independent unit of work* that a build performs, such as compiling classes, creating a JAR, generating Javadoc, or publishing archives to a repository.
 
-image::writing-tasks-1.png[]
+++++
+<p style="text-align: center;">
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/FB5muea-2Z0?si=XcoZGjrw0KD5FSIJ&rel=0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Comment:
I would use bigger video size by default

Reply:
I'll play with this. This was recommended by YouTube. I just need to make sure it resizes properly.